contrib/windows/julia-manifest.xml

```diff
@@ -7,6 +7,11 @@
       </requestedPrivileges>
     </security>
   </trustInfo>
+  <dependency>
+    <dependentAssembly>
+      <assemblyIdentity type='win32'  name='Microsoft.Windows.Common-Controls' version='6.0.0.0'     processorArchitecture='*' publicKeyToken='6595b64144ccf1df' language='*' />
```

Comment:
Hmm, I tried to insert the manifest manually as explained further down in the Gitter messages (using Resource Hacker) and actually made no difference.
So, one more reason to wait for Nenad's answer (Red main developer)